base/shared/src/main/scala/scalaz/core/eq.scala

```diff
@@ -15,5 +16,7 @@ trait EqSyntax {
   implicit final class ToEqOps[A](a: A) {
     private[core] type equal
     def ===(f: A)(implicit ev: Eq[A]): Boolean = macro meta.Ops.nia_1[equal]
+    private[core] type notEqual
```

Comment:
Curious, is this just used for the macro to "find" the alphanumerically-named method rather than one by the same name?

Reply:
Yep.